springmvc-raml-parser/src/main/java/com/phoenixnap/oss/ramlapisync/generation/rules/basic/MethodParamsRule.java

```diff
@@ -193,7 +194,10 @@ protected JVar paramObjects(ApiActionMetadata endpointMetadata, CodeModelHelper.
 		if (!RamlActionType.PATCH.equals(endpointMetadata.getActionType())) {
 			// skip Valid annotation for PATCH actions since it's a partial
 			// update so some required fields might be omitted
-			param.annotate(Valid.class);
+            if(counter < 1) {

```

Comment:
If I change it to < 2, and param.annotate(Valid.class) is called twice, a resulting class will not have Valid as an import. Valid will be added directly to methods as a fully qualified name:
```
    @RequestMapping(value = "", method = RequestMethod.PUT)
    public ResponseEntity<?> updateUsers(
        @javax.validation.Valid
        @org.springframework.web.bind.annotation.RequestBody
        UpdateUsersRequest updateUsersRequest);
```
(this only happens for raml08, search for a line like: `AbstractRuleTestBase.RAML = RamlLoader.loadRamlFromFile("raml08issue.raml");`)